include/zephyr/bluetooth/bluetooth.h

```diff
@@ -69,36 +69,59 @@ extern "C" {
  */
 #define BT_LE_LOCAL_SUPPORTED_FEATURES_SIZE 8
 
-/** Opaque type representing an advertiser. */
+/** @brief Opaque type representing an advertiser. */
 struct bt_le_ext_adv;
 
-/** Opaque type representing an periodic advertising sync. */
+/** @brief Opaque type representing an periodic advertising sync. */
 struct bt_le_per_adv_sync;
 
-/* Don't require everyone to include conn.h */
+/** @brief Don't require everyone to include conn.h */
 struct bt_conn;
 
-/* Don't require everyone to include iso.h */
+/** @brief Don't require everyone to include iso.h */
 struct bt_iso_biginfo;
 
-/* Don't require everyone to include direction.h */
+/** @brief Don't require everyone to include direction.h */
 struct bt_df_per_adv_sync_iq_samples_report;
 
+/**
+ * @brief Info of the advertising sent event.
+ *
+ * @note bt_le_ext_adv_sent_info is used in @ref bt_le_ext_adv_cb.
+ */
 struct bt_le_ext_adv_sent_info {
 	/** The number of advertising events completed. */
 	uint8_t num_sent;
 };
 
+/**
+ * @brief Info of the advertising connected event.
+ *
+ * @note bt_le_ext_adv_connected_info is used in @ref bt_le_ext_adv_cb.
+ */
 struct bt_le_ext_adv_connected_info {
 	/** Connection object of the new connection */
 	struct bt_conn *conn;
 };
 
+/**
+ * @brief Info of the advertising scanned event.
+ *
+ * @note bt_le_ext_adv_scanned_info is used in @ref bt_le_ext_adv_cb.
```

Comment:
Generally missing `@ref`s for structs (assuming that we want a clickable link. If not, then perhaps it could be simpler by just being

```suggestion
 * @note Used in @ref bt_le_ext_adv_cb.
```